desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -121,6 +122,7 @@ public SignedOfferView(OpenOfferManager openOfferManager, XmrWalletService xmrWa
 
     private Offer fetchOffer(SignedOffer signedOffer) {
         Trade trade = tradeManager.getTrade(signedOffer.getOfferId());
+
```

Comment:
Remove added whitespace.

Reply:
I got an error by simply creating an offer and attempting to publish the offer's penalty tx:

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/13068859/220358059-ae6101ae-a363-4241-9037-bbf416994d8b.png">

The problem is, the offer may or may not be available on the books, so we can't retrieve the offer.

However, it's not necessary. The confirmation popup should only display details which are readily available in the `SignedOffer`. The offer id and penalty tx id is fine to start.

So fetching the offer should be removed only fields available in SignedOffer should be displayed.

In a next iteration, we'll add additional fields to `SignedOffer` like the mining fee and penalty amount so they'll be easy to display in the popup.

Reply:
Only rely on fields in `SignedOffer` to show the confirmation popup. The original offer may or may not be available on the books, and so we should not rely on it at at all.